File: src/main/java/at/hannibal2/skyhanni/utils/jsonobjects/BingoJson.java
package at.hannibal2.skyhanni.utils.jsonobjects;

import com.google.gson.annotations.Expose;

import java.util.List;
import java.util.Map;

public class BingoJson {
    @Expose
    public Map<String, BingoTip> bingo_tips;

    public static class BingoTip {
        @Expose
        public String difficulty;

        @Expose
        public List<String> note;
    }
}

File: src/main/java/at/hannibal2/skyhanni/data/jsonobjects/repo/BingoJson.kt
package at.hannibal2.skyhanni.data.jsonobjects.repo

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class BingoJson(
    @Expose @SerializedName("bingo_tips") val bingoTips: Map<String, BingoData>,
)

data class BingoData(
    @Expose val difficulty: String,
    @Expose val note: List<String>,
    @Expose val guide: List<String>,
    @Expose val found: String?,
)
